OneSignalSDK/build.gradle

```diff
@@ -3,7 +3,7 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
         maven { url 'https://maven.google.com' }
```

Comment:
we could use google() instead.
https://stackoverflow.com/questions/46467561/difference-between-google-and-maven-url-https-maven-google-com